src/ir_Coolix.cpp

```diff
@@ -170,11 +218,10 @@ uint8_t IRCoolixAC::getTemp() {
   uint8_t i;
   for (i = 0; i < kCoolixTempRange; i++)
     if (kCoolixTempMap[i] == code) return kCoolixTempMin + i;
-  return kCoolixUnknown;  // Not a temp we expected.
+  return kCoolixTempMax;  // Not a temp we expected.
```

Comment:
Might this hide when we are getting a value we don't (yet) understand, and perhaps should?